### PR TITLE
Plugins should support i18n. This commit adds the basics, but the engine...

### PR DESCRIPTION
**Pull request by bkearney**
...ers need to start translating code

### CODE REVIEW

File: hammer_cli_csv.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = %w(lib)
 
   spec.add_dependency('hammer_cli_katello')
-
+  spec.add_dependency("gettext", "~> 2.0")
```
Comment (komidore64) [inline hammer_cli_csv.gemspec]: so this is needed during runtime, as well as running `rake gettext:find` ?

Comment (bkearney) [inline hammer_cli_csv.gemspec]: used for both, as other plugins keep it in Gemfile outside of the test block

**Comment by komidore64** [comment]
ack

File: hammer_cli_csv.gemspec (hunk 1 above)
Comment (komidore64) [inline hammer_cli_csv.gemspec]: sweet!

**Pull request merged**